dev/org.eclipse.codewind.ui/src/org/eclipse/codewind/ui/internal/wizards/BindProjectWizard.java

```diff
@@ -102,11 +103,16 @@ public boolean performFinish() {
 			projectPath = projectPage.getProjectPath();
 		}
 
+		final String type = projectTypePage.getType().getId();
+		ProjectSubtypeInfo projectSubtype = projectTypePage.getSubtype();
+		final String subtype = (projectSubtype == null) ? null : projectSubtype.id;
```

Comment:
The variable subtype is not used anywhere.  Does it not need to be passed to the bind request?

Reply:
Never mine - I see you have this as a todo.